aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -210,6 +211,8 @@ def __init__(
 class Route:
     """Internally used Route Configuration"""
 
+    _middleware_built = False
```

Comment:
not sure I can infer by the name - what's the intent?

Reply:
local class variable to indicate the middleware stack has already been built (middleware stack construction only occurs once - once the handler is wrapped - doesn't need to happen again - its a bad name.